lib/salus/scanners/slither.rb

```diff
@@ -0,0 +1,55 @@
+require 'json'
+require 'salus/scanners/base'
+
+# Slither (https://github.com/crytic/slither) is a Python-based static
+# analysis tool for Solidity smart contracts
+
+module Salus::Scanners
+  class Slither < Base
+    include Salus::Formatting
+
+    def should_run?
+      !!@repository.sol_file_present?
```

Comment:
What does `!!` mean? I have never seen this syntax before.

Reply:
Because [sol_file uses wildcard matching](https://github.com/coinbase/salus/pull/607/files#diff-50c2ad55bc3977fdf09bc45a06d90d7febb43141bed71c7f8c49d503b44a90c8R44), `sol_file_present?` returns an array of `.sol` filepaths. `!!` lets us coerce the value of the array from truthy (non-empty array) or falsey (empty array) to `true` or `false`, respectively.

Reply:
[The `should_run?` tests](https://github.com/coinbase/salus/pull/607/files#diff-dadba0278118c6a057fb1d5f6ed9e58c5cce981b84578da134b4ce7dfff09068R5-R20) demonstrate this behavior.